packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -69,7 +69,9 @@ function ListViewBlockSelectButton(
 			>
 				<ListViewExpander onClick={ onToggleExpanded } />
 				<BlockIcon icon={ blockInformation?.icon } showColors />
-				<BlockTitle clientId={ clientId } maximumLength={ 35 } />
+				<span className="block-editor-list-view-block__title">
```

Comment:
Should this be `block-editor-list-view-block-select-button__title`?

Reply:
Yes. Good spot!